### PR TITLE
test/peer_states: add test for probing back to health with slower than period

### PR DESCRIPTION
**Pull request by jcorbin**
Per @kriskowal 's observation in #1007

### CODE REVIEW

**Comment by jcorbin** [comment]
TBD: whether this is actually what we want, my intuition says we should have some sort of decay so that it takes more slow-rate probes to get back to good... but maybe this is actually what we want.

File: node/test/peer_states.js [hunk 1]
```diff
@@ -252,6 +252,68 @@ testSetup('consecutive success during unhealthy periods restores health', {}, fu
...
+
+        function checkBefore(done) {
+            peer.channel.timers.advance(1000);
+            assert.equals(peer.state.shouldRequest(), 1.0, 'expected probe score before try ' + trial);
+            assert.equals(peer.state.type, 'tchannel.unhealthy', 'unhealthy before try ' + trial);
+
+            done();
+        },
+
+        cluster.send('glad'),
```
Comment (Raynos) [inline node/test/peer_states.js]: Why does a single glad not transition it back to healthy ?

**Comment by Raynos** [comment]
lgtm.

**Comment by kriskowal** [comment]
woot. ask and you shall indeed receive!

**Pull request merged**